### ISSUE TITLE
Cannot change return type of existing function

### ISSUE THREAD

**Issue by mhov** [PostgresCompare/postgrescompare-issues#7]
Using Version 1.0.52

if the target db has something like 
`CREATE OR REPLACE FUNCTION my_func() RETURNS TABLE (col1 text) AS ...`
and the source has something like
`CREATE OR REPLACE FUNCTION my_func() RETURNS TABLE (col1 text, col2 text) AS ...`

The compare will categorize it (correctly) as "Create or Replace Function" but the deploy SQL will only be
```
CREATE OR REPLACE FUNCTION my_func() RETURNS TABLE (col1 text, col2 text) AS ..
```
which causes `42P13: cannot change the return type of existing function` because the return type is different
![image](https://user-images.githubusercontent.com/2199565/59132776-f5624280-892a-11e9-8548-c6e7d31b69df.png)

I'd expect to drop the existing function before creating the new one
```
DROP FUNCTION my_func();
CREATE FUNCTION my_func() RETURNS TABLE (col1 text, col2 text) AS ..   ;
```

**Comment by sql-migrate** [comment]
@mhov working on this one

**Comment by neilanderson** [comment]
Fixed in v1.1.10 ty @mhov
